WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -267,21 +266,12 @@
             </ControlTemplate>
           </Setter>
         </Style>
-
       </TreeDataGrid.Styles>
     </TreeDataGrid>
     <Panel IsVisible="{Binding IsTransactionHistoryEmpty}">
       <c:HistoryPlaceholderPanel Margin="{StaticResource HistoryTablePlaceholderPanelMargin}"
                                  RowHeight="{StaticResource HistoryTableRowHeight}" />
-      <Panel Background="{DynamicResource RegionColor}" Opacity="0.75" />
-      <Grid RowDefinitions="26,1*,1.5*"
-            IsVisible="{Binding IsTransactionHistoryLoaded}"
-            Margin="{StaticResource HistoryTablePlaceholderPanelMargin}">
-        <c:InfoMessage VerticalAlignment="Center"
-                       HorizontalAlignment="Center"
-                       Grid.Row="1"
-                       Content="Transaction history is empty" />
-      </Grid>
+      <Panel Background="{DynamicResource RegionColor}" Opacity="0.90" />
```

Comment:
This is not correct and probably wasn't correct previously.
It also fades the header and everything...

This `Panel` should be removed and the opacity should be set on `HistoryPlaceholderPanel` control.

And it can be simplified furthermore.

PR:
```xml
    <Panel IsVisible="{Binding IsTransactionHistoryEmpty}">
      <c:HistoryPlaceholderPanel Margin="{StaticResource HistoryTablePlaceholderPanelMargin}"
                                 RowHeight="{StaticResource HistoryTableRowHeight}" />
      <Panel Background="{DynamicResource RegionColor}" Opacity="0.90" />
    </Panel>
```

Simplfied:
```xml
<c:HistoryPlaceholderPanel Margin="{StaticResource HistoryTablePlaceholderPanelMargin}"
                               RowHeight="{StaticResource HistoryTableRowHeight}"
                               IsVisible="{Binding IsTransactionHistoryEmpty}"
                               Opacity="0.1" />
```

Posting like this, maybe easier to see what to replace with what.
Adjust the opacity as you think.

Reply:
With those changes the header is now clickable and you can sort columns, which seems weird IMO.
Should we just go with that?

![9](https://user-images.githubusercontent.com/52379387/198019705-e3a47a67-999b-481e-85b4-0269873dd430.PNG)

Reply:
You can fix it by adding this line:
```
IsHitTestVisible="{Binding !IsTransactionHistoryEmpty}"
```
To the `TreeDataGrid` control.

This basically makes the control unclickable until the list is empty.

Reply:
> which seems weird IMO

☝️